inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -25,22 +26,22 @@ class SourceParameters:
 
```

Comment:
**Best Practice**
**Suggestion:** Consider using a more specific type than list for sources to ensure type consistency across your codebase.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
@dataclass(kw_only=True, frozen=True)
class ApplicationSourceList:
    name: str
    sources: List[str]
```